src/unix_process/unix_process_runtime.c

```diff
@@ -72,13 +85,69 @@ void halt(char *format, ...)
     exit(2);
 }
 
+#ifndef __aarch64__
 static heap allocate_tagged_region(heap h, u64 tag)
 {
     u64 size = 256 * MB;
     void *region = mmap(pointer_from_u64(tag << VA_TAG_OFFSET),
                         size, PROT_READ | PROT_WRITE, MAP_FIXED | MAP_ANON | MAP_PRIVATE, -1, 0);
     return (heap)create_id_heap(h, h, u64_from_pointer(region), size, 1);
 }
+#else
+struct tagheap {
+    struct heap h;
+    heap mh;
+    u64 vtag;
+};
+
+static void tag_dealloc(heap h, u64 a, bytes s)
+{
+    struct tagheap *th = (struct tagheap *)h;
+    tag_debug("%s: tag %d, a 0x%lx, s 0x%lx\n", __func__, th->vtag >> VA_TAG_OFFSET, a, s);
```

Comment:
tag_debug() already adds `__func__` at the beginning of the output, so another `__func__` here is redundant. The same applies to the other uses of tag_debug() in this file.